src/map/ai/helpers/pathfind.cpp

```diff
@@ -362,6 +362,7 @@ void CPathFind::StepTo(const position_t& pos, bool run)
 
     float stepDistance = (speed / 10) / 2;
     float distanceTo   = distance(m_POwner->loc.p, pos);
+    float diff_y = pos.y - m_POwner->loc.p.y;
```

Comment:
From CI: Styling needid so the "=" aligns with the ones above.

Reply:
Oops, thank you. I will await other feedback before making a commit.